styles.css

```diff
@@ -0,0 +1,65 @@
+*{
+  box-sizing:border-box;
+}
+
+
+.u-list-reset {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.u-list-inline li {
+  display: inline-block;
+  padding-left: 10px;
+  padding-left: 10px;
+}
+
+body {
+  background-color:hsl(234,18.5%,10.6%)
```

Comment:
As a way to set some defaults for your whole HTML document, I recommend that every `body` ruleset include the following:
- `background` or `background-color` ✅ You've done this.
- `color`
- `font-family`

You can always change these defaults on individual elements or specific classes elsewhere in your stylesheet, but adding them here creates a solid foundation and will make sure your document displays as expected across browsers.